metaci/release/utils.py

```diff
@@ -143,9 +144,9 @@ def send_start_webhook(project_config, release, role, config_item):
     payload = {
         "implementation_step_id": f"{implementation_step_id}",
     }
-    token = jwt_session()
+    token = jwt_for_webhook()
     response = requests.post(
-        f"http://0.0.0.0:8001/implementation_step_id/{implementation_step_id}/start/",
+        f"http://{settings.METACI_RELEASE_WEBHOOK_URL}/implementation_step_id/{implementation_step_id}/start/",
```

Comment:
The setting should include https.

Also I meant for you to make the same change on line 184